python/GafferUI/EditScopeUI.py

```diff
@@ -380,6 +390,58 @@ def __userNodes( editScope ) :
 		nodes = Gaffer.Metadata.nodesWithMetadata( editScope, "editScope:includeInNavigationMenu" )
 		return [ n for n in nodes if n.ancestor( Gaffer.EditScope ).isSame( editScope ) ]
 
+	def __dropNode( self,  event ) :
+
+		node = self.__inputNode()
+		if node is None :
+			return None
```

Comment:
I wonder if it's worth removing this check and dealing with the "nothing is being viewed" case in the same way we do the "editscope is not upstream" case, with a popup message?

Reply:
Sounds good, I've done that in an additional commit: c6bda13

Reply:
...and a quick fixup to ensure the highlighting doesn't linger: d929ce8